### PR TITLE
Add Zyxel OS cfg_firewall_acl_get

### PR DESCRIPTION
**Pull request by elavaud**
##### ISSUE TYPE
 - New Template Pull Request

##### COMPONENT
Zyxel, no specific OS, cfg firewall acl get

### CODE REVIEW

**Comment by elavaud** [comment]
It is failing trying to update pyparsing to 2.4.7... 
Not related to the PR. Any way to relaunch the Travis CI?

File: ntc_templates/templates/zyxel_os_cfg_firewall_acl_get.textfsm [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Value PORTS ((\d+|Any)(:\d+)?)
+Value ACTION (Accept|Drop)
+
+Start
+  ^Index\s+(Enable\s+)?Order\s+Name\s+Source\sIP\s+Destination\sIP\s+Service\s+Action\s*$$ -> FIREWALLTable
+  ^\s*$$
+  ^. -> Error
+
+FIREWALLTable
+  ^${INDEX}\s+(${ENABLE}\s+)?${ORDER}\s+${NAME}\s+${SOURCE_IP}\s+${DESTINATION_IP}\s+${PROTOCOL}(:Any-->${PORTS})?\s+${ACTION}\s*$$ -> Record
```
Comment (itdependsnetworks) [inline ntc_templates/templates/zyxel_os_cfg_firewall_acl_get.textfsm:17]: "Any" seems unnecessarily restrictive

File: ntc_templates/templates/index [hunk 2]
```diff
@@ -506,3 +506,5 @@ watchguard_firebox_show_arp.textfsm, .*, watchguard_firebox, sh[[ow]] arp
 
 yamaha_show_environment.textfsm, .*, yamaha, sh[[ow]] en[[vironment]]
 yamaha_show_ip_route.textfsm, .*, yamaha, sh[[ow]] ip ro[[ute]]
+
+zyxel_os_cfg_firewall_acl_get.textfsm, .*, zyxel_os, cfg firewall(\s|_)acl get
```
[the remark this answers is not in the document]
Comment (elavaud) [reply ntc_templates/templates/index:510]: I thought so but can not find the device with a space... So let's keep the underscore only.

**Comment by elavaud** [comment]
Oh, now I understand why the `(\s|_)`.
The command contains an underscore. However I don't know how to escape the underscore in the test name file.
Now it looks for a template for "cfg firewall acl get" instead of "cfg firewall_acl get".

How should I do?

**Comment by elavaud** [comment]
Grouped in #1142

**Pull request closed without merging**